clippy_lints/src/utils/attrs.rs

```diff
@@ -15,7 +15,10 @@ pub enum DeprecationStatus {
 
 pub const BUILTIN_ATTRIBUTES: &[(&str, DeprecationStatus)] = &[
     ("author", DeprecationStatus::None),
-    ("cyclomatic_complexity", DeprecationStatus::None),
+    (
+        "cyclomatic_complexity",
+        DeprecationStatus::Replaced("cognitive_complexity"),
+    ),
```

Comment:
So if I understand this list correctly, you'll need to add an entry for `cognitive_complexity` with `DeprecationStatus::None`, otherwise you'll get complaints about an unknown attribute when you start using `#[clippy::cognitive_complexity(..)]`

Reply:
Ahh! I see. Thank you :D